docs/giskard_hub/index.rst

```diff
@@ -0,0 +1,12 @@
+Giskard Hub
+===============
+
+.. toctree::
```

Comment:
Same as for integrations :

Should we use :hidden: here, and use cards instead ?